LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 speakeasy-sdks
```

Comment:
@amruth-opal should we update this copyright?

Reply:
Is that how that works? We made it from the speakeasy codegen